share/src/main/resources/assets/wdl/lang/zh_CN.lang

```diff
@@ -0,0 +1,523 @@
+wdl.gui.wdl.entityOptions.name=实体选项...
+wdl.gui.wdl.entityOptions.description=控制保存哪些实体，以及这些实体的跟踪距离。
+wdl.gui.wdl.gameruleOptions.name=游戏规则...
+wdl.gui.wdl.gameruleOptions.description=选择各个游戏规则。
+wdl.gui.wdl.backupOptions.name=备份选项...
+wdl.gui.wdl.backupOptions.description=控制保存世界后如何备份。
+wdl.gui.wdl.messageOptions.name=消息选项...
+wdl.gui.wdl.messageOptions.description=控制聊天中显示的消息。
+wdl.gui.wdl.savedChunks.name=已保存的区块...
+wdl.gui.wdl.savedChunks.description=以可视化查看当前、以前和何时保存了哪些区块
+wdl.gui.wdl.permissionsInfo.name=权限信息...
+wdl.gui.wdl.permissionsInfo.description=此服务器权限配置的信息，以及请求新权限的表单。
+wdl.gui.wdl.about.name=关于World Downloader...
+wdl.gui.wdl.about.description=查看您当前安装的WDL和扩展的信息。
+wdl.gui.wdl.updates.name=更新...
+wdl.gui.wdl.updates.description=查看版本列表及当前版本，并配置下一次更新的条件。
+wdl.gui.wdl.updates.hasNew.name=更新... (§6有新版可用！§r)
+wdl.gui.wdl.updates.hasNew.description=查看版本列表及当前版本，配置下一次更新的条件，下载最新更新。
+
+wdl.gui.world.title=World Downloader mod: %s 的世界选项
+wdl.gui.world.allowCheats.true=允许作弊：开
+wdl.gui.world.allowCheats.false=允许作弊：关
+wdl.gui.world.gamemode.keep=游戏模式：不做更改
+wdl.gui.world.gamemode.creative=游戏模式：创造
+wdl.gui.world.gamemode.survival=游戏模式：生存
+wdl.gui.world.gamemode.hardcore=游戏模式：极限生存
+wdl.gui.world.time.keep=时间：不做改变
+wdl.gui.world.time.23000=时间：日出
+wdl.gui.world.time.0=时间：上午
+wdl.gui.world.time.6000=时间：中午
+wdl.gui.world.time.11500=时间：傍晚
+wdl.gui.world.time.12500=时间：日落
+wdl.gui.world.time.18000=时间：午夜
+wdl.gui.world.weather.keep=天气：不做改变
+wdl.gui.world.weather.sunny=天气：晴
+wdl.gui.world.weather.rain=天气：雨
+wdl.gui.world.weather.thunderstorm=天气：雷暴
+wdl.gui.world.spawn.auto=出生点：自动
+wdl.gui.world.spawn.player=出生点：玩家位置
+wdl.gui.world.spawn.xyz=出生点：
+wdl.gui.world.setSpawnToCurrentPosition=当前位置
+wdl.gui.world.allowCheats.description=控制下载的世界是否允许命令（例如 /gamemode）。
+wdl.gui.world.gamemode.description=控制下载的世界中使用何种游戏模式。
+wdl.gui.world.time.description=控制下载的世界的时间。
+wdl.gui.world.weather.description=控制下载的世界的天气。
+wdl.gui.world.spawn.description=控制下载的世界的出生点。\n\n§l不做更改§r：使用服务器的出生点。\n§l玩家位置§r：使用下载结束时玩家所在位置。\n§lXYZ§r：允许指定精确的坐标。
+wdl.gui.world.setSpawnToCurrentPosition.description=将您现在的位置设置为出生点。
+wdl.gui.world.spawnPos.description=使用文本框指定精确的出生点坐标。\n\n此文本框指定的坐标为 %s 。
+
+wdl.gui.player.title=World Downloader mod: %s 的玩家选项
+wdl.gui.player.health.keep=生命：不做改变
+wdl.gui.player.health.20=生命：加满
+wdl.gui.player.hunger.keep=饥饿：不做改变
+wdl.gui.player.hunger.20=饥饿：加满
+wdl.gui.player.position.keep=玩家位置：不做改变
+wdl.gui.player.position.xyz=玩家位置：
+wdl.gui.player.setPositionToCurrentPosition=当前位置
+wdl.gui.player.health.description=此选项修改在下载的世界中玩家的生命。\n\n§l不做改变§r：使用下载结束时的生命值。\n§l加满§r：将玩家生命设置为最大值。
+wdl.gui.player.hunger.description=此选项修改在下载的世界中玩家的饥饿。\n\n§l不做改变§r：使用下载结束时的饥饿值。\n§l加满§r：将玩家饥饿设置为最大值。
+wdl.gui.player.position.description=此选项可修改在下载的世界中玩家的位置。\n\n§l不做改变§r：使用下载结束时的玩家的位置\n§lXYZ§r：允许指定精确的坐标。
+wdl.gui.player.setPositionToCurrentPosition.description=将位置设置为您当前的位置。
+wdl.gui.player.positionTextBox.description=使用文本框指定精确的出生点坐标。\n\n此文本框指定的坐标为 %s 。
+
+wdl.gui.generator.title=World Downloader mod:: %s 的生成选项
```

Comment:
This looks like a typo; I assume it should only be 1 `:`?

Reply:
yep, I have changed it.